engine/flink/components/dev-table/src/main/scala/pl/touk/nussknacker/engine/flink/table/sink/SqlSinkFactory.scala

```diff
@@ -0,0 +1,74 @@
+package pl.touk.nussknacker.engine.flink.table.sink
+
+import pl.touk.nussknacker.engine.api.component.ProcessingMode
+import pl.touk.nussknacker.engine.api.context.ValidationContext
+import pl.touk.nussknacker.engine.api.context.transformation.{
+  DefinedEagerParameter,
+  NodeDependencyValue,
+  SingleInputDynamicComponent
+}
+import pl.touk.nussknacker.engine.api.definition.{NodeDependency, ParameterWithExtractor, TypedNodeDependency}
+import pl.touk.nussknacker.engine.api.process.{Sink, SinkFactory}
+import pl.touk.nussknacker.engine.api.{NodeId, Params}
+import pl.touk.nussknacker.engine.flink.table.sink.SqlSinkFactory.ValueParamName
+import pl.touk.nussknacker.engine.flink.table.utils.SqlComponentFactory
+import pl.touk.nussknacker.engine.flink.table.utils.SqlComponentFactory.getSelectedTableUnsafe
+import pl.touk.nussknacker.engine.flink.table.{SqlDataSourcesDefinition, TableDefinition}
+import pl.touk.nussknacker.engine.util.parameters.SingleSchemaBasedParameter
+
+object SqlSinkFactory {
+  val ValueParamName = "Value"
+}
+
+class SqlSinkFactory(definition: SqlDataSourcesDefinition) extends SingleInputDynamicComponent[Sink] with SinkFactory {
+
+  override type State = TableDefinition
+
+  // TODO: add non-raw value parameters
+  private val RawValueParam = ParameterWithExtractor.lazyMandatory[AnyRef](ValueParamName)
+
+  private val tableNameParam: ParameterWithExtractor[String] =
+    SqlComponentFactory.buildTableNameParam(definition.tableDefinitions)
+  private val TableNameParamName = tableNameParam.parameter.name
+
+  override def contextTransformation(context: ValidationContext, dependencies: List[NodeDependencyValue])(
+      implicit nodeId: NodeId
+  ): this.ContextTransformationDefinition = {
+    case TransformationStep(Nil, _) =>
+      NextParameters(
+        parameters = RawValueParam.parameter :: tableNameParam.parameter :: Nil,
```

Comment:
From the user perpsective IMO it is more natural to provide a table and after that value. We follow such an order in other components such as kafka

Reply:
done :+1: